fitbenchmarking/fitbenchmarking
# Removing windows systests

#### Description of Work

Fixes #644 

Removes windows systests

#### Testing Instructions

1. Check that the windows tests are completely removed

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
